manifests/scripts/metrics-dashboard/run.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# -----------------
+
+# Using OpenShift's Prometheus instance requires us to enable user workload monitoring via the openshift-monitoring ConfigMap:
+# https://docs.openshift.com/container-platform/4.11/monitoring/enabling-monitoring-for-user-defined-projects.html)
+# - BUT, the way I'm doing it here is a brittle hack :P
+kubectl apply -f prometheus/enable-user-workload-monitoring.yaml
```

Comment:
It's probably possible to use the `openshift.io/cluster-monitoring` label on the various namespaces to enable prometheus scraping without having to enable user workload monitoring.  But it may not be worth doing right now.

Reply:
@jopit That would definitely be much easier... does OpenShift support using this label for user-defined projects? E.g. perhaps it's something that technically works, but isn't for users to use?

I only ask because I don't see any reference to this label on [user monitoring page](https://docs.openshift.com/container-platform/4.11/monitoring/enabling-monitoring-for-user-defined-projects.html), or any other references in the docs that indicate it can be used this way, which makes me wonder :thinking: .

Reply:
Right it's not for users, its for Red Hat. But we are Red Hat.  Not sure what the actual restrictions on using it are, the PM's said we should use it for the gitops operator to set up monitoring for it's argo-cd instances.

Reply:
Ah, gotcha, yup, that makes sense.